CSSOM_view module page (#38276)
mdn/content: files/en-us/web/css/cssom_view/index.md

- {{domxref("MediaQueryList")}} events
  - {{domxref("MediaQueryList/change_event", "change")}}

### Glossary

> - {{glossary("viewport")}}

Reviewer: It feels like it would be good to use the same capitalization on these as the titles of pages.

Also, the "Visual Viewport" page uses different capitalization than the "Layout viewport" page, in both title and slug. It would be good to make these as consistent as possible.

> - {{glossary("viewport")}}
> - {{glossary("layout viewport")}}
> - {{glossary("visual viewport")}}

Author: ```suggestion
- {{glossary("Viewport")}}
- {{glossary("Layout viewport")}}
- {{glossary("Visual viewport")}}
```

  - {{domxref("Element.scrollTo", "scrollTo()")}}
  - {{domxref("Element.scrollTop", "scrollTop")}}
  - {{domxref("Element.scrollWidth", "scrollWidth")}}
- {{domxref("HTMLElement")}}
  - {{domxref("HTMLElement.offsetHeight", "offsetHeight")}}
  - {{domxref("HTMLElement.offsetLeft", "offsetLeft")}}
  - {{domxref("HTMLElement.offsetParent", "offsetParent")}}
  - {{domxref("HTMLElement.offsetTop", "offsetTop")}}
  - {{domxref("HTMLElement.offsetWidth", "offsetWidth")}}
- {{domxref("HTMLImageElement")}}
  - {{domxref("HTMLImageElement.x", "x")}}
  - {{domxref("HTMLImageElement.y", "y")}}
- {{domxref("Range")}}
  - {{domxref("Range.getBoundingClientRect", "getBoundingClientRect()")}}
  - {{domxref("Range.getClientRects", "getClientRects()")}}
- {{domxref("MouseEvent")}}
  - {{domxref("MouseEvent.clientX", "clientX")}}
  - {{domxref("MouseEvent.clientY", "clientY")}}
  - {{domxref("MouseEvent.offsetX", "offsetX")}}
  - {{domxref("MouseEvent.offsetY", "offsetY")}}
  - {{domxref("MouseEvent.pageX", "pageX")}}
  - {{domxref("MouseEvent.pageY", "pageY")}}
  - {{domxref("MouseEvent.screenY", "screenY")}}
  - {{domxref("MouseEvent.x", "x")}}
  - {{domxref("MouseEvent.y", "y")}}

> This module define geometric utility methods which will apply to the {{domxref("Text")}}, {{domxref("Text")}}, {{domxref("Text")}}, and {{domxref("CSSPseudoElement")}} interfaces. These `GeometryUtils` features are not yet implemented in any browsers.

Reviewer: ```suggestion
This module defines geometric utility methods that will apply to the {{domxref("Text")}}, {{domxref("Text")}}, {{domxref("Text")}}, and {{domxref("CSSPseudoElement")}} interfaces. These `GeometryUtils` features are not yet implemented in any browsers.
```

Reviewer: Also, "{{domxref("Text")}}, {{domxref("Text")}}, {{domxref("Text")}}" needs updating ;-)

## Guides

- [Coordinate systems](/en-US/docs/Web/CSS/CSSOM_View/Coordinate_Systems)

>   - : A guide to the coordinate systems used to specify the position of a location in a display context, whether that context is a window on a monitor, a viewport on a mobile device, or a position on a sheet of paper when printing.

Reviewer: ```suggestion
  - : A guide to the coordinate systems used to specify a position in a display context such as a window on a monitor, a viewport on a mobile device, or a position on a sheet of paper when printing.
```

>   - : A guide to the coordinate systems used to specify the position of a location in a display context, whether that context is a window on a monitor, a viewport on a mobile device, or a position on a sheet of paper when printing.

Author: ```suggestion
  - : The coordinate systems used to specify a position in a display context such as a window on a monitor, a viewport on a mobile device, or a position on a sheet of paper when printing.
```

- [Viewport concepts](/en-US/docs/Web/CSS/CSSOM_View/Viewport_Concepts)

>   - : The concept of the viewport — what it is, its impact in terms of CSS, SVG, and mobile devices — and the difference between the visual viewport and the layout viewport.

Reviewer: I'd change this to the same text as the introduction on the actual page, as the grammar is better:

```suggestion
  - : This article explains the concept of the viewport — what it is, its impact in terms of CSS, SVG, and mobile devices — and differentiates between the visual viewport and the layout viewport.
```

>   - : The concept of the viewport — what it is, its impact in terms of CSS, SVG, and mobile devices — and the difference between the visual viewport and the layout viewport.

Author: ```suggestion
  - : The concept of the viewport — what it is, its impact in terms of CSS, SVG, and mobile devices — and differentiates between the visual viewport and the layout viewport.
```

## Guides

- [Coordinate systems](/en-US/docs/Web/CSS/CSSOM_View/Coordinate_Systems)
  - : The coordinate systems used to specify a position in a display context such as a window on a monitor, a viewport on a mobile device, or a position on a sheet of paper when printing.
- [Viewport concepts](/en-US/docs/Web/CSS/CSSOM_View/Viewport_Concepts)

>   - : The concept of the viewport — what it is, its impact in terms of CSS, SVG, and mobile devices — and differentiates between the visual viewport and the layout viewport.

Reviewer: This change is wrong; if you used "differentiates" like this, you'd have to change the sentence structure to something like

> This article explains the concept of the viewport — what it is and its impact in terms of CSS, SVG, and mobile devices — and differentiates between the visual viewport and the layout viewport.

>   - : The concept of the viewport — what it is, its impact in terms of CSS, SVG, and mobile devices — and differentiates between the visual viewport and the layout viewport.

Author: ```suggestion
  - : The concept of the viewport — what it is, its impact in terms of CSS, SVG, and mobile devices — and the difference between the visual viewport and the layout viewport.
```
